components/VerifyOptions.yml

```diff
@@ -19,7 +19,7 @@ components:
       properties:
         verificationMethod:
           type: string
-          description: The URI of the verificationMethod used for the proof. Default assertionMethod URI.
+          description: The URI of the verificationMethod used for the proof. If omitted a default verification method will be used.
```

Comment:
I'm wondering if we want to allow overriding the verificationMethod in the proof. It feels like a dangerous thing to do? I'm just wondering out loud right now... don't have a super strong feeling on this yet.

Reply:
I agree this feels sensitive. We should merge this PR here and then create a new issue to discuss removing `verificationMethod` from `VerifyOptions`.

Reply:
I disagree that our default approach to identified security concerns is "merge it and discuss". People are implement to these APIs and we don't want to inject potential vulnerabilities into our specs.